kratos/containers/geometry_container.h

```diff
@@ -55,13 +55,15 @@ class GeometryContainer
     typedef std::size_t IndexType;
     typedef std::size_t SizeType;
 
+    typedef typename TGeometryType::Pointer GeometryTypePointer;

```

Comment:
Would you please replace the `GeometryTypePointer` with `GeometryPointerType`. The latter is more coherent with the rest of the type definitions which have Type as sufix.

Reply:
Sure. Makes sense. I changed it.

Thanks for the review!